.github/workflows/integration.yml

```diff
@@ -34,13 +26,12 @@ jobs:
       run: cargo test --all
 
     - name: Build optimized binary
-      run: cargo build --release --locked
+      run: cargo build --release
 
-    - name: Upload build artifacts
-      uses: actions/upload-artifact@master
+    - uses: actions/upload-artifact@master
```

Comment:
I recommend using `actions/upload-artifact@v2` instead of `master` for stability.